mypy/build.py

```diff
@@ -2936,6 +2936,12 @@ def process_graph(graph: Graph, manager: BuildManager) -> None:
 
     fresh_scc_queue: List[List[str]] = []
 
+    if manager.options.baseline_file:
+        res = manager.errors.load_baseline(Path(manager.options.baseline_file))
+        if not res and manager.options.baseline_file != defaults.BASELINE_FILE:
+            print("Baseline file not found")
+            raise Exception("Baseline file not found")
```

Comment:
why does it print the error as well as raising it? won't raising it also print it?